desktop/renderer-app/package.json

```diff
@@ -67,7 +67,6 @@
     "css-loader": "^5.2.4",
     "dotenv-flow": "^3.2.0",
     "dotenv-flow-webpack": "^1.1.0",
-    "edit-json-file": "^1.6.0",
```

Comment:
missing changes to the `yarn.lock` file.

We should run: `yarn remove edit-json-file` in the project directory.
Instead of editing the dependencies in _package.json_ directly

Reply:
The yarn.lock file is still unchanged